tibber/__init__.py

```diff
@@ -196,7 +201,7 @@ async def update_info(self) -> None:
             self._all_home_ids += [home_id]
             if not (subs := _home.get("subscriptions")):
                 continue
-            if subs[0].get("status", "ended").lower() == "running":
+            if subs[0].get("status", "ended") is not None and subs[0].get("status", "ended").lower() == "running":
```

Comment:
This is not related to fetching websocket url but i was getting errors when testing on a home with no subs